src/nxdoc/nuxeo-server/administration/elasticsearch-setup.md

```diff
@@ -473,14 +473,20 @@ This page provides several configuration use cases for [Elasticsearch](https://w
 
 ### Elasticsearch Supported Versions
```

Comment:
it's because the compatibility matrix used to be a big nice table and now it's just replaced by
> Elasticsearch compatibility details can be found on the [Compatibility Matrix]({{page page='compatibility-matrix#elasticsearch'}}) page.